openmetadata-docs/content/v1.1.0/quick-start/local-deployment.md

```diff
@@ -145,6 +145,7 @@ pip3 install --upgrade pip setuptools
 ```
 
 ### 5. Install the OpenMetadata Python module using pip
+On macOS, make sure to install XCode prior to running the following command:
```

Comment:
nit - if we ask to install something, either add the installation command or a link to external docs that show how to get this installed. Thanks